src/fundus/publishers/tr/ntvtr.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_date_parsing,
+)
+
+
+class NTVTRParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector(
+            "#contentBodyArea > div.category-detail-content-inner > div.content-news-tag-selector > p"
```

Comment:
This newspaper seems to support a `summary` as well. Look for `category-detail-sub-title`

Further, also `topics` and `authors` are supported. You can find them is the LD or meta respectively.